File: app/src/main/java/com/icerrate/popularmovies/view/movies/catalog/MoviesCatalogActivity.java
package com.icerrate.popularmovies.view.movies.catalog;

import android.os.Bundle;
import android.view.Window;

import com.icerrate.popularmovies.R;
import com.icerrate.popularmovies.view.common.BaseActivity;

/**
 * @author Ivan Cerrate.
 */

public class MoviesCatalogActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_movies_catalog);
        setTitle(getString(R.string.title_activity_movies));
        setNavigationToolbar(false);
        if (savedInstanceState == null) {
            MoviesCatalogFragment moviesCatalogFragment = MoviesCatalogFragment.newInstance();
            replaceFragment(R.id.content, moviesCatalogFragment);
        }
    }
}

File: app/src/main/java/com/icerrate/popularmovies/view/movies/catalog/MoviesCatalogActivity.kt
package com.icerrate.popularmovies.view.movies.catalog

import android.os.Bundle
import com.icerrate.popularmovies.R
import com.icerrate.popularmovies.databinding.ActivityMoviesCatalogBinding
import com.icerrate.popularmovies.view.common.BaseActivity
import dagger.hilt.android.AndroidEntryPoint

/**
 * @author Ivan Cerrate.
 */
@AndroidEntryPoint
class MoviesCatalogActivity : BaseActivity<ActivityMoviesCatalogBinding>(
    ActivityMoviesCatalogBinding::inflate
) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setSupportActionBar(binding.toolbar)
        binding.toolbar.title = getString(R.string.title_activity_movies)
        setNavigationToolbar(false)
        if (savedInstanceState == null) {
            val moviesCatalogFragment = MoviesCatalogFragment.newInstance()
            replaceFragment(R.id.content, moviesCatalogFragment)
        }
    }
}